independent-projects/arc/runtime/src/main/java/io/quarkus/arc/impl/CollectionHelpers.java

```diff
@@ -13,16 +13,20 @@
         //do not instantiate
     }
 
+    @SuppressWarnings("unchecked")
     static <T> Set<T> toImmutableSmallSet(Set<T> set) {
         if (set == null)
             return null;
         switch (set.size()) {
             case 0:
-                return Collections.emptySet();
+                return Set.of();
```

Comment:
I didn't know this was actually better

Reply:
I had never seen this method :) Seems it's new in Java 9.
But also it's a constant in both cases apparently... I suppose you prefer it because of style Martin ?

Reply:
It's not better, I just wanted to be consistent here... ;-)